x11-drivers/nvidia-drivers/nvidia-drivers-460.39.ebuild

```diff
@@ -1,44 +1,38 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
+
 inherit desktop flag-o-matic linux-info linux-mod multilib-minimal \
 	nvidia-driver portability systemd toolchain-funcs unpacker udev
 
-AMD64_FBSD_NV_PACKAGE="NVIDIA-FreeBSD-x86_64-${PV}"
 AMD64_NV_PACKAGE="NVIDIA-Linux-x86_64-${PV}"
-ARM_NV_PACKAGE="NVIDIA-Linux-armv7l-gnueabihf-${PV}"
+ARM64_NV_PACKAGE="NVIDIA-Linux-aarch64-${PV}"
```

Comment:
I'm tad confused here. All ebuilds add `ARM64_NV_PACKAGE` but none of them actually uses it.

Reply:
correct, this is just preparing everything for `arm64` support, which I'll add soon